recipes/tiledb-py/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = "0.1.0" %}
+
+package:
+  name: tiledb-py
+  version: {{ version }}
+
+source:
+  url: https://github.com/TileDB-Inc/TileDB-Py/archive/{{ version }}.tar.gz
+  sha256: 8223e468f6c01bd82be4fdfee2a3b58af205b6ddd0f812fc4000b983304c8f66
+
+build:
+  number: 0
+  skip: true  # [win or py2k]
+  script: python setup.py install --tiledb=$PREFIX --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
```

Comment:
Does this not also depend on `setuptools-scm`?

Reply:
Not for this version, for later versions it will be a dependency.